File: DSA PRACTICE/LECTURE5/MCQ1.java
//MCQ:01 on Unary Operator

public class MCQ1
{
	public static void main(String[] args) {
		int a,b=1;
		a=10;
		if(++a)
		{
			System.out.print("b: "+b);
		}
		else 
		{
			System.out.print("++b: "+b);	
		}
	}
}

/*
OUTPUT: MCQ1.java:8: error: incompatible types: int cannot be converted to boolean
                if(++a)
*/

File: DSA PRACTICE/LECTURE5/MCQ1.cpp
//MCQ:01 on Unary Operator
#include<iostream>
using namespace std;

int main()
{
	int a,b=1;
	a=10;
	if(++a) // true=1 and false=0
	{
		cout<<"b: "<<b<<endl;
	}
	else
	{
		cout<<"++b: "<<++b<<endl;
	}
	return 0;
}

/*
OUTPUT: b=1
*/